examples/addsvc/server/grpc/encode_decode.go

```diff
@@ -0,0 +1,41 @@
+package grpc
+
+import (
+	"github.com/go-kit/kit/examples/addsvc/pb"
+	"github.com/go-kit/kit/examples/addsvc/server"
+	"golang.org/x/net/context"
```

Comment:
Please separate go-kit from third-party imports, and put third-party above. i.e.

```go
import (
	"golang.org/x/net/context"

	"github.com/go-kit/kit/examples/addsvc/pb"
	"github.com/go-kit/kit/examples/addsvc/server"
)
```